bases/overlay-common/root/.s3cfg.sample

```diff
@@ -29,15 +27,19 @@ recursive = False
 recv_chunk = 256000
 reduced_redundancy = False
 send_chunk = 256000
-signature_v2 = True
+signature_v2 = False
```

Comment:
What's the rationale behind this? (I'm pretty newbie on s3 related matters)

Reply:
it's a request authentification method. v4 is better :)

Reply:
Sounds good!